core/base/inc/TROOT.h

```diff
@@ -95,6 +95,9 @@ namespace ROOT {
    void DisableImplicitMT();
    Bool_t IsImplicitMTEnabled();
    UInt_t GetImplicitMTPoolSize();
+
+   /// Get the number of system CPUs.
+   UInt_t GetNCPUs();
```

Comment:
(Especially since) this is not configurable, this routine belongs in TSystem.  Thanks.

Reply:
Sure!